### PR TITLE
Feature/email results of chomp session

### PR DESCRIPTION
**Pull request by mxng**
## Why

To have email to our users

## What

This pull request introduces the following:

 * Email to user telling them their account is created
 * Email to user telling them their Chomp session is created

## How to Test

- Create an account, and check the email of the account
- Create a chomp session, and check the email again for another email

## Others

Try to go to my repo and see if `rails s` works.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CLOUDINARY_URL=
+GMAIL_ADDRESS=
```
Comment (TeeWhyKay) [inline .env.sample:2]: do i need to add your gmail address and password to heroku config?

File: app/mailers/chomp_session_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.chomp_session_mailer.create_chomp.subject
+  #
+  def create_chomp
+    @chomp_session = params[:chomp_session]
+
+    mail(
+      to:       @chomp_session.user.email,
```
Comment (TeeWhyKay) [inline app/mailers/chomp_session_mailer.rb:12]: is there a check if the user is a guest but did not specify his email?

File: .env.sample (hunk 1 above)
Comment (mxng) [reply .env.sample:2]: think so, else you can make your own and change the files with my email inside into your email (there is a sender: <my email> somewhere)

File: app/mailers/chomp_session_mailer.rb (hunk 2 above)
Comment (mxng) [reply app/mailers/chomp_session_mailer.rb:12]: The user who created the chomp session needs an account so an email will already be there

Comment (TeeWhyKay) [reply app/mailers/chomp_session_mailer.rb:12]: okay 🤩

**Review by TeeWhyKay — APPROVED** [review]
great job with the email part!

**Pull request merged**